README.md

```diff
@@ -433,15 +460,15 @@ user experience:
    browser to a backend endpoint that
    redirects the browser to a URL with the intent
    scheme it is possible to use an existing AS
-   without modifications.
+   without modifications - though compared to App Links redirecting via the web browser is a poorer UX and leaves a tab open in the user's browser.
```

Comment:
This method is only used to come back to the RP app in the app2web case. So we are already in the browser and do not have to open a new browser tap. From a UX point of view, there is no difference to App Links here.

Reply:
I would like to remove this addition because it was my intention to use the intent:// scheme rather for web2app than for app2app redirection. Maybe we have to somehow clarify this here but it is already in the section ``Web2App Solutions on Android``. Or have I misunderstood your point here?

Reply:
I think we can point out that it has a poorer UX and leaves a tab open compared to applinks (because both points are true), but we can still say that this seems to be the best method. i.e. we're pointing out that the solution is a compromise that achieves the best outcome that can be achieved.

Reply:
But didn't an Applink behave exactly the same? In which case does an intent URL leave a tab open where an Applink would not leave one open?

Reply:
Sorry, you're right - this is web2app, I was confusing it with one of the discussions we had at some point about using web interstitials in app2app. I've removed my change.